sphinx/source/docs/dev_guide/notes.rst

```diff
@@ -44,6 +44,9 @@ There are several environment variables that can be useful for developers:
 * ``BOKEH_PRETTY`` --- Whether to emit "pretty printed" JSON
     Accepted values are ``yes``/``no``, ``true``/``false`` or ``0``/``1``.
 
+* ``BOKEH_PY_LOGGING`` -- Whether to enable logging by default.
```

Comment:
needs three `-` to make an "em" dash like the others also the others don't have a period (probably should but for now just want to be consistent)